### PR TITLE
adding frontend setup

### PR DESCRIPTION
**Pull request by KarishmaSinghChauhan**

### CODE REVIEW

File: frontend/src/router/index.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+        //     name: "SignUp",
+        //     component: SignUp,
+        //     meta: {
+        //         goToHome: true,
+        //     }
+        // },
+        // {
+        //     path: "/verify",
+        //     name: "Verify",
+        //     component: Verify,
```
Comment (sdumka) [inline frontend/src/router/index.js:60]: Can we remove these and add these in the next change

File: frontend/src/router/index.js [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+        //     name: "MyShops",
+        //     component: MyShops,
+        //     meta: {
+        //         requiresAuth: true,
+        //     },
+        // },
+    ],
+});
+
+// router.beforeEach((to, from, next) => {
```
Comment (sdumka) [inline frontend/src/router/index.js:91]: same

File: frontend/src/router/index.js [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+        //     },
+        // },
+        // {
+        //     path: "/my-shops",
+        //     name: "MyShops",
+        //     component: MyShops,
+        //     meta: {
+        //         requiresAuth: true,
+        //     },
+        // },
```
Comment (sdumka) [inline frontend/src/router/index.js:87]: same

File: frontend/src/main.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+import firebase from "firebase";
+import App from './App.vue'
+import './registerServiceWorker'
+import router from './router'
+import store from './store'
+
+Vue.config.productionTip = false
+
+let app = "";
+const config = {
```
Comment (sdumka) [inline frontend/src/main.js:11]: Can you add the link to firebase console from where you obtained below values.

File: frontend/src/components/HelloWorld.vue [hunk 5]
```diff
@@ -0,0 +1,119 @@
+<template>
```
Comment (sdumka) [inline frontend/src/components/HelloWorld.vue:1]: Can we rename this class to be more specific to our project?

File: frontend/src/main.js (hunk 4 above)
Comment (KarishmaSinghChauhan) [reply frontend/src/main.js:11]: Where exactly do you want me to add the firebase console link?

File: frontend/src/router/index.js (hunk 1 above)
Comment (KarishmaSinghChauhan) [reply frontend/src/router/index.js]: Done

File: frontend/src/router/index.js (hunk 2 above)
Comment (KarishmaSinghChauhan) [reply frontend/src/router/index.js]: Done

File: frontend/src/router/index.js (hunk 3 above)
Comment (KarishmaSinghChauhan) [reply frontend/src/router/index.js]: Done

File: frontend/src/components/HelloWorld.vue (hunk 5 above)
Comment (KarishmaSinghChauhan) [reply frontend/src/components/HelloWorld.vue]: File not required. Hence removed.

File: frontendservice/.gitignore [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+.vscode
+*.suo
+*.ntvs*
+*.njsproj
+*.sln
+*.sw?
+
+#
+public/img/*
+package-lock.json
```
Comment (priytanvani) [inline frontendservice/.gitignore:26]: Nit. Add newline at the end?

Comment (KarishmaSinghChauhan) [reply frontendservice/.gitignore]: Added newline at the end of .gitignore

**Pull request merged**